stm32/libcanard/bxcan/src/bxcan.c

```diff
@@ -0,0 +1,753 @@
+/// This software is distributed under the terms of the MIT License.
+/// Copyright (c) 2016-2020 UAVCAN Development Team.
+/// Authors: Pavel Kirienko <pavel.kirienko@zubax.com>, Tom De Rybel <tom.derybel@robocow.be>
+
+#include "bxcan.h"
+#include "bxcan_registers.h"
+#include <assert.h>
+#include <string.h>
+
+/// Configure the maximum interface index for the bxCAN hardware available in your MCU.
+/// Must be set to either 0 (only CAN1) or 1 (CAN1 and CAN2).
+#define BXCAN_MAX_IFACE_INDEX 1U
+
+/// Configure the system core clock frequency in Hz.
+/// Only used by the busy wait in waitMSRINAKBitStateChange().
+#define BXCAN_BUSYWAIT_DELAY_SYSTEM_CORE_CLOCK 80000000UL
```

Comment:
Ok, looks like a good idea.
Question: should I add one like this for the BXCAN_MAX_IFACE_INDEX? It would force people to think about it and set it to the right value, so the tests for the interface number work more reliable.

Reply:
> should I add one like this for the BXCAN_MAX_IFACE_INDEX? It would force people to think about it and set it to the right value, so the tests for the interface number work more reliable.

Yes, that would make sense.

Reply:
Done.